pkg/clickhouse/clickhouse.go

```diff
@@ -567,3 +577,24 @@ func (ch *ClickHouse) GetPartitions(database, table string) (map[string][]metada
 	}
 	return result, nil
 }
+
```

Comment:
there is main part of refactoring, i just wrap all query related methods inside `package clickchouse`
and log it in clickhouse-backup output
and try to don't log it inside clickhouse-server (via pass log_queries=0)